pages/clubs.jsx

```diff
@@ -323,6 +324,11 @@ var ModalAddOrChangeYourClub = React.createClass({
       'name', 'website', 'description', 'location', 'latitude', 'longitude'
     ));
     var validationErrors = validateClub(clubState);
+
+    if (!this.props.club && !this.state.hasReadFactSheet) {
```

Comment:
The `!this.props.club` is checking to see if we're the "add your club" form or the "change your club" form--we don't show the checkbox in the latter.